rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -162,6 +163,7 @@ def get_balances(
                     result[address].append({
                         'id': nft.token_identifier,
                         'name': nft.name,
+                        'collection_name': nft.collection.name,
```

Comment:
Collection is marked as optional in the NFT class. Shouldn't you check if it has collection first?